locations/spiders/essentiel_antwerp.py

```diff
@@ -0,0 +1,29 @@
+import json
+import re
+
+import scrapy
+
+from locations.items import Feature
+
+
+class EssentielAntwerpSpider(scrapy.Spider):
+    name = "essentiel_antwerp"
+    item_attributes = {"brand": "Essentiel-Antwerp"}
+    start_urls = ["https://www.essentiel-antwerp.com/be_fr/storelocator"]
+
+    def parse(self, response):
+        data = response.xpath(".").get()
```

Comment:
There is `response.text` for when it's a text response, or `response.body` for none text.
```suggestion
        data = response.text
```